src/lwc/geLabelService/geLabelService.js

```diff
@@ -60,6 +60,7 @@ import geButtonFormFieldsModalDeleteSectionAndFields from '@salesforce/label/c.g
 import geButtonTemplatesTabCreateTemplate from '@salesforce/label/c.geButtonTemplatesTabCreateTemplate';
 import geErrorCompleteThisField from '@salesforce/label/c.geErrorCompleteThisField';
 import geErrorExistingTemplateName from '@salesforce/label/c.geErrorExistingTemplateName';
+import geErrorDonorTypeValidation from '@salesforce/label/c.geErrorDonorTypeValidation';
```

Comment:
do we want to add this to this service class here (?) 
 - even if i'm not using it and the label is only going to be used for HGE, since it's a custom validation that only applies to HGE form.

> i was planning to use this service but i didn't after seeing all the unnecessary stuff that gets loaded, instead i just called the label and the format method from uiCommon

Reply:
Yes, if the labels you're trying to use as part of HGE aren't in the service class, feel free to add them.